coordinator/src/routes/prescription/post-process-message.ts

```diff
@@ -15,6 +15,8 @@ export default [
       async (requestPayload: Bundle, request: Hapi.Request, responseToolkit: Hapi.ResponseToolkit) => {
         const spineRequest = translator.convertFhirMessageToSpineRequest(requestPayload)
         const spineResponse = await requestHandler.send(spineRequest, request.logger)
+        request.log("audit", {messageType: "Inbound FHIR message", payload: requestPayload})
```

Comment:
wont this contain potentially sensitive information?

Reply:
same as with the other logs

Reply:
Yea, thats the only issue with this approach. It _should_ be ok, given the only people with access to production logs are security cleared

Reply:
happy to approve if we are sure people with access to production logs are security cleared

Reply:
There was a discussion on Slack on 06/01 about user data in logs, think we decided it all had to be removed? We've merged in a PR to remove  messages from other log events